src/tests/sys_tests.rs

```diff
@@ -90,5 +90,14 @@ pub mod test_sys {
         assert_eq!(child_cage.cwd.read(),cage.cwd.read())
     }
 
+    pub fn ut_lind_exit() {
+        // Since exit function is heavily used and tested in other syscalls and their tests 
+        // We only perform preliminary checks for checking the sanity of this syscall
+        lindrustinit(0);
+        let cage = interface::cagetable_getref(1);
+        // Call the exit call
+        assert_eq(cage.exit_syscall(EXIT_SUCCESS),EXIT_SUCCESS); 
+        lindrustfinalize(); 
+    }
```

Comment:
I wonder if it would be necessary to test some error cases, like exit a cage twice?

Reply:
Like fork and exec - exit is also handled by the NaCL runtime which ensures that exit is not called twice on a cage. This was similar to when we create a new cage using fork we don't check for the process to be already existing or not because that is handled by the NaCL runtime

Reply:
would be great to document this.